### DISCUSSION TITLE
Onde drive cask doesn't follow the naming pattern of other microsoft formulas

### DISCUSSION THREAD

**Discussion by elizeusdsantos** [Homebrew/discussions#4905] [Casks] [answered]
### Output of `brew config`

```shell
not related
```

### Output of `brew doctor`

```shell
not related
```

### Description of issue

We have available several Microsoft products through casks. All of them are following the pattern _microsoft-{**product_name**}_. However, that's not the case of OneDrive.

### Relevant casks

onedrive

**Comment by miccal** [comment, marked as answer]
The naming follows our token reference: https://docs.brew.sh/Cask-Cookbook#token-reference 

Note that the `app` bundle for Microsoft OneDrive is called `OneDrive.app`, while for Microsoft Word (for example) it is called `Microsoft Word.app`.
